datadog_checks_base/datadog_checks/base/utils/db/utils.py

```diff
@@ -183,6 +183,27 @@ def resolve_db_host(db_host):
     return db_host
 
 
+def get_agent_host_tags():
```

Comment:
Let's also add this function to datadog_agent stub in `datadog_checks_base/datadog_checks/base/stubs/datadog_agent.py` and add a new test case

Reply:
Good call, I added it there and added a test